gandalf.js

```diff
@@ -3,115 +3,105 @@
  *
```

Comment:
SCRIPT-This line seems to be longer than our standard line length. Could you split it up for better readability?